RELEASE_NOTES.md

```diff
@@ -1,3 +1,72 @@
+- Allow configuring the size of the AST cache with a new --ast-cache-size command line option. Thanks [@Babar](http://github.com/Babar)
+
+### ChefSpec 7.4.0
+
+ChefSpec has been updated to 7.4 with better support stubbing commands and a new `policyfile_path` configuration option for specifying the path to the PolicyFile.
+
+### kitchen-dokken 2.7.0
+
+kitchen-dokken has been updated to 2.7.0 with new options for controlling how containers are setup and pulled. You can now disable user namespace mode when running privileged containers with a new `userns_host` config option. There is also a new option `pull_chef_image` (true/false) to control force pulling the chef image on each run to check for newer images. This now defaults to true so that testing on latest and current always actually mean latest and current. See the [kitchen-digitalocean readme](https://github.com/test-kitchen/kitchen-digitalocean/blob/master/README.md) for `kitchen.yml` config examples.
+
+### kitchen-digitalocean 0.10.4
+
+kitchen-digitalocean has been updated to 0.10.4 with support for new distros and additional configuration options for instance setup. You can now control the default DigitalOcean region systems are spun up in using a new `DIGITALOCEAN_REGION` env var. You can still modify the region in the driver section of your `kitchen.yml` file if you'd like, and the default region of `nyc1` has not changed. This release also adds slug support for `fedora-29`, `fedora-30`, and `ubuntu-19`. Finally if you'd like to monitor your test instances there is new `monitoring` configuration option in the `kitchen.yml` driver section that allows enabling DigitalOcean's instance monitoring. See the [kitchen-dokken readme](https://github.com/someara/kitchen-dokken/blob/master/README.md) for `kitchen.yml` config examples.
+
+### knife-vsphere 3.0.0
+
+knife-vsphere has been updated to 3.0. This new version adds support for specifying the `bootstrap_template` when creating new VMs. This release also improves how the plugin finds VM hosts in order to support hosts in nested directories.
+
+## End of Ubuntu 14.04 support
+
+Ubuntu 14.04 entered the end-of-life phase April 30, 2019. Since this version of Ubuntu is now end of life we have stopped building packages for Ubuntu 14.04. If you rely on Ubuntu 14.04 in your environment we highly recommend upgrading your host to Ubuntu 16.04 or 18.04.
+
+### Security Updates
+
+###  curl 7.65.1
+
+- CVE-2019-5435: Integer overflows in curl_url_set
+- CVE-2019-5436: tftp: use the current blksize for recvfrom()
+- CVE-2018-16890: NTLM type-2 out-of-bounds buffer read
+- CVE-2019-3822: NTLMv2 type-3 header stack buffer overflow
+- CVE-2019-3823: SMTP end-of-response out-of-bounds read
+- CVE-2019-5443: Windows OpenSSL engine code injection
+
+### cacerts 5-11-2019 release
+
+- Our cacert bundle has been updated to the 5-11-2019 bundle which adds four additional CAs.
```

Comment:
```suggestion
- Our `cacert` bundle has been updated to the 5-11-2019 bundle, which adds four additional CAs.
```

Reply:
Formatting for `cacert` may be incorrect -- so please correct if I am wrong -- but the comma before "which" is definitely needed.

Reply:
seems fine this way. Thanks